File: app/src/main/java/com/example/tmorimatsu/newsapp/NewsViewHolder.java
package com.example.tmorimatsu.newsapp;

import android.support.annotation.NonNull;
import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

public class NewsViewHolder extends RecyclerView.ViewHolder {

    ImageView imageView;
    TextView descriptionText;
    TextView publishText;
    View row;

    NewsViewHolder(@NonNull View itemView) {
        super(itemView);
        imageView = itemView.findViewById(R.id.image_row);
        descriptionText = itemView.findViewById(R.id.content_row);
        publishText = itemView.findViewById(R.id.publish_row);
        row = itemView.findViewById(R.id.row);
    }
}

File: app/src/main/java/com/example/tmorimatsu/newsapp/NewsViewHolder.kt
package com.example.tmorimatsu.newsapp

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.ImageView
import android.widget.TextView

class NewsViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    val imageView = itemView.findViewById<ImageView>(R.id.image_row)
    val descriptionText = itemView.findViewById<TextView>(R.id.content_row)
    val publishText = itemView.findViewById<TextView>(R.id.publish_row)
    val row = itemView.findViewById<View>(R.id.row)
}